css/index.css

```diff
@@ -27,14 +43,26 @@ h2 {
     padding: 1.5rem;
     background-color: lightgray;
 }
+#leave_message {
+    display: flex;
+    border: 1px solid #000;
+    background: cornsilk;
+    
+}
+.form {
```

Comment:
I don't know if you need a new class, `.form`, I would either:
- Change the styling for `form` element directly in index.css, like you already do with `nav` and other HTML elements
```
form {
   ....
}
```

- or, move the styling into the `#leave_message` which is what you have above.

Reply:
ok, did that! much cleaner